File: src/BukkitPE/level/particle/InstantEnchantParticle.php
<?php
namespace BukkitPE\level\particle;

use BukkitPE\math\Vector3;

class InstantEnchantParticle extends GenericParticle{
	public function __construct(Vector3 $pos){
		parent::__construct($pos, Particle::TYPE_MOB_SPELL_INSTANTANEOUS);
	}
}

File: src/main/java/net/BukkitPE/level/particle/InstantEnchantParticle.java
package net.BukkitPE.level.particle;

import net.BukkitPE.math.Vector3;

/**
 * Created on 2015/11/21 by xtypr.
 * Package net.BukkitPE.level.particle in project BukkitPE .
 */
public class InstantEnchantParticle extends GenericParticle {
    public InstantEnchantParticle(Vector3 pos) {
        super(pos, Particle.TYPE_MOB_SPELL_INSTANTANEOUS);
    }
}
